#A message that tells the joint to move a specified speed.
#Positive is counter clockwise
string[] joint_names
float64[] positions
float64[] velocities
float64[] accelerations

